src/plugins/intel_gna/gna_device.hpp

```diff
@@ -97,6 +97,19 @@ class GNADeviceHelper {
         }
     }
 
+    bool debugLogEnabled = false;
```

Comment:
1. please move fields members which shouldn't be which should be accessed outside to private part.
2. please move fields members which should be public to the common part on top with others publics (or move to private and add set method).

Reply:
Done

Reply:
all are public